docs/mongocli/command/mongocli-iam-users-invite.txt

```diff
@@ -69,7 +69,7 @@ Options
    * - --password
      - string
      - false
-     - User's password.
+     - Password for the database user.
```

Comment:
this is similar to the problem we had before, we have reused the constant in more than one context and in this case it doesn't apply, this is an Atlas User and not database user, similar to before it'll require to create a new constant with the new copy specific to database users

Reply:
@gssbzn Thanks, I updated it to use a new variable and changed the description for this one back to be more generic.